catalog.yml

```diff
@@ -238,3 +238,5 @@
 - repo_url: https://github.com/greenelab/optimizer-manuscript
   html_url: https://greenelab.github.io/optimizer-manuscript/
   preprint_citation: doi:10.1101/2023.06.26.546586
+- repo_url: https://github.com/Adafede/thesis-manuscript
+  journal_citation: 10.13097/archive-ouverte/unige:165668
```

Comment:
Nice. The `html_url` key is required as much of the metadata about the paper is taken from there. Since you don't have a manubot generated HTML online, it might work to use the published version like https://archive-ouverte.unige.ch/unige:165668 as the `html_url`. Do you want to try that here?

Reply:
Good suggestion, happy to try!